### DISCUSSION TITLE
OpenRouteService routing

### DISCUSSION THREAD

**Discussion by ToToGSARider** [devemux86/cruiser#563] [Q&A] [answered]
Hallo zusammen, ich habe bei mir in Cruiser im Routing den Open Route Service aktiviert. Wie kann man dort die Profile beeinflussen? Wenn ich eine Route von mir zu einem anderen Punkt fahren möchte, werde Fahrradwege bzw. Feldwege verwendet. Wie kann man dem Route Service denn sagen, dass er nur Hauptstraßen verwenden soll bzw. Straßen die man auch befahren darf?

Hallo zusammen, ich habe bei mir in Cruiser im Routing den Open Route Service aktiviert. Wie kann man dort die Profile beeinflussen? Wenn ich eine Route von mir zu einem anderen Punkt fahren möchte, werde Fahrradwege bzw. Feldwege verwendet. Wie kann man dem Route Service denn sagen, dass er nur Hauptstraßen verwenden soll bzw. Straßen die man auch befahren darf?

**Comment by devemux86** [comment, marked as answer]
@ToToGSARider 

OpenRouteService has separate options in the Cruiser settings, have you checked them?
They are not as detailed as in BRouter (offline).

OpenRouteService hat separate Optionen in den Cruiser-Einstellungen, haben Sie diese überprüft?
Sie sind nicht so detailliert wie in BRouter (offline).

<img src="https://github.com/devemux86/cruiser/assets/3484020/a169a67e-2cfa-4f1d-acd9-c9d00f1cd8a5" width="50%"/>

**Reply by ToToGSARider** [reply]
Die Einstellung Schnell scheint das Problem zu lösen, Dankeschön :-) 
Werde da mal rumspielen und schauen was er da so macht.

**Comment by devemux86** [comment]
@ToToGSARider 

You can also try Valhalla, it has more options.

Sie können auch Valhalla ausprobieren, es bietet mehr Möglichkeiten.

<img src="https://github.com/devemux86/cruiser/assets/3484020/cdf16e3d-b01e-4c01-9bc1-e62693f5f407" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/317ccae7-ca62-4945-88d3-d7e5fc60c41b" width="50%"/>

**Reply by ToToGSARider** [reply]
Kann man dort auch die Profile aktivieren / auswählen, die auf der Internetseite vorgeschlagen werden?
![image](https://github.com/devemux86/cruiser/assets/128730865/0aab97cc-4d3f-4254-ae3d-adf0164541cd)
das was dort passiert, ist nämlich echt gut.

**Reply by devemux86** [reply]
@ToToGSARider 

You can select the motorcycle profile in Cruiser.
Then Valhalla offers the same options for motorcycle or scooter.

Sie können das Motorradprofil in Cruiser auswählen.
Dann bietet Valhalla die gleichen Optionen für Motorrad oder Motorroller.

**Reply by ToToGSARider** [reply]
Super, Dankeschön.
